src/pages/Home/Home.js

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Navbar, Footer } from "../../components";
+
+import "./home.css";
+
+export const Home = () => {
+  return (
+    <>
+      <div className={`parallax-wrapper`}>
```

Comment:
Suggestion - If you won't be adding any variable in className, then you can use single quotes here instead of template strings.

Reply:
Yup, i actually forgot about the backticks here. Thank you